### PR TITLE
[#67] Twitter and VK don't receive the page title when the script's placed before it

### PR DESCRIPTION
**Pull request by vitkarpov**
Define `defaults` when the DOM's ready.

### CODE REVIEW

**Comment by vitkarpov** [comment]
@iamakulov

**Comment by iamakulov** [comment]
(Sorry, I’m still not here and really busy these days. I’ll get to all this stuff on the next week.)

**Comment by vitkarpov** [comment]
That's okay :)

File: source/likely.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ var likely = require('./index.js');
 var history = require('./history');
 
 window.addEventListener('load', function () {
+    likely.defaults = likely.getDefaultOptions();
```
Comment (iamakulov) [inline source/likely.js]: As far as I see, `likely.defaults` is used in only one place – [`index.js:23`](https://github.com/ilyabirman/Likely/blob/master/source/index.js#L23). Is setting it necessary?

Comment (vitkarpov) [inline source/likely.js]: Do you think we can use it on demand, just to call `getDefaultOptions()`? Or to drop it at all?

Comment (iamakulov) [inline source/likely.js]: I believe we can drop it at all.

Comment (iamakulov) [inline source/likely.js]: Like, I mean, we definitely need some default options, but there’s no need even in `getDefaultOptions()`.

Comment (vitkarpov) [inline source/likely.js]: You mean we can define defaults just right near the https://github.com/ilyabirman/Likely/blob/master/source/index.js#L23 , where it's used?

Comment (iamakulov) [inline source/likely.js]: Yep, this seems the best solution to me.

**Comment by vitkarpov** [comment]
@iamakulov fixed according review:

- get rid of `likely.defaults` and use it on demand, 'cause there's only only place to use

**Comment by iamakulov** [comment]
Thanks!

**Pull request merged**